h2o-py/h2o/explanation/_explain.py

```diff
@@ -1291,6 +1294,18 @@ def ice_plot(
     if frame.type(column) == "string":
         raise ValueError("String columns are not supported!")
 
+    distribution = _get_distribution(model)
+    if distribution not in ['bernoulli', 'binomial'] and (binary_score_form == "logodds" or binary_score_form == "probability"):
```

Comment:
We also have `quasibinomial` and `fractionalbinomial` "distributions".  And also `custom` distribution which as far as I know can be applied on binomial classification as well. 

Maybe something like `model._model_json["output"]["model_category"] == "Binomial"` would work better? @maurever do you know if that would work correctly even for the `custom` distribution with binomial response?

Reply:
I thought the `model_category` is set based on response distribution, so it is definitely better to decide with it. Good point @tomasfryda .